doc/nrf/shortcuts.txt

```diff
@@ -249,3 +249,6 @@
    When using any other DK, the Toolchain Manager installation is recommended for the |NCS| v1.9.x and earlier versions.
 
 .. |filter_samples_by_board| replace:: If you want to list samples available for one or more specific boards, `use the nRF Connect for Visual Studio Code extension to filter them <Browse samples_>`_.
+
+.. |54H_engb_2_8| replace:: When using the nRF54H20 DK Engineering B (from v0.8.0 to 0.8.2), you must build samples and applications using the board revision 0.8.0 with the ``<board>@<revision>`` syntax.
```

Comment:
Isn't this note applicable for matter, nRF desktop, Machine learning applications as well, since these application also have nRF54H20 support?

Reply:
Added also to the applications' doc